<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Part1Dialog</class>
 <widget class="QDialog" name="Part1Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>88</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>Execute!</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>enter a</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="a_value"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>enter i</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="i_value"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>enter j</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="j_value"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>enter k</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="k_value"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>enter m</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_value"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>175</width>
     <height>252</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="text">
         <string>Binary representation of A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="binaryRepresentation">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="text">
         <string>K-th bit of A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="k_bit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="text">
         <string>A after (re)set k-th bit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="aAfterSetK">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="text">
         <string>A after swap i-th and j-th bits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="aAfterSwap">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="text">
         <string>A after reset m bits</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="AAfterResetM">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
